### ISSUE TITLE
Error parsing parameters: Failed to load network spec

### ISSUE THREAD

**Issue by prestonvanloon** [ConsenSys/web3signer#494]
```
Error parsing parameters: Failed to load network spec: /path/to/e2e_config.yaml
```

I'm receiving the above error without any reason or justification. The docs mention the flag value can be "Predefined network configuration. Accepts a predefined network name, or file path or URL to a YAML configuration file. See the Ethereum 2.0 specification for examples." 

```
Version = web3signer/v21.10.5/linux-x86_64/-ubuntu-openjdk64bitservervm-java-11
```

This is the file I'm trying to load. It's a modified version of the [minimal config](https://github.com/ethereum/consensus-specs/blob/180855b579830c3f757bb63eda3bf3793f37cd90/configs/minimal.yaml). 

```yaml
# e2e config

# Extends the minimal preset
PRESET_BASE: 'minimal'


# Transition
# ---------------------------------------------------------------
# TBD, 2**256-2**10 is a placeholder
TERMINAL_TOTAL_DIFFICULTY: 115792089237316195423570985008687907853269984665640564039457584007913129638912
# By default, don't use these params
TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615



# Genesis
# ---------------------------------------------------------------
# [customized]
MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 256 # Override for e2e tests
# Jan 3, 2020
MIN_GENESIS_TIME: 1578009600
# Highest byte set to 0x01 to avoid collisions with mainnet versioning
GENESIS_FORK_VERSION: 0x00000001
# [customized] Faster to spin up testnets, but does not give validator reasonable warning time for genesis
GENESIS_DELAY: 10 # Override for e2e tests


# Forking
# ---------------------------------------------------------------
# Values provided for illustrative purposes.
# Individual tests/testnets may set different values.

# Altair
ALTAIR_FORK_VERSION: 0x01000001
ALTAIR_FORK_EPOCH: 6 # Override for e2e
# Bellatrix
BELLATRIX_FORK_VERSION: 0x02000001
BELLATRIX_FORK_EPOCH: 18446744073709551615
# Sharding
SHARDING_FORK_VERSION: 0x03000001
SHARDING_FORK_EPOCH: 18446744073709551615


# Time parameters
# ---------------------------------------------------------------
# [customized] Faster for testing purposes
SECONDS_PER_SLOT: 10 # Override for e2e tests
# 14 (estimate from Eth1 mainnet)
SECONDS_PER_ETH1_BLOCK: 2 # Override for e2e tests
# 2**8 (= 256) epochs
MIN_VALIDATOR_WITHDRAWABILITY_DELAY: 256
# [customized] higher frequency of committee turnover and faster time to acceptable voluntary exit
SHARD_COMMITTEE_PERIOD: 4 # Override for e2e tests
# [customized] process deposits more quickly, but insecure
ETH1_FOLLOW_DISTANCE: 4 # Override for e2e tests


# Validator cycle
# ---------------------------------------------------------------
# 2**2 (= 4)
INACTIVITY_SCORE_BIAS: 4
# 2**4 (= 16)
INACTIVITY_SCORE_RECOVERY_RATE: 16
# 2**4 * 10**9 (= 16,000,000,000) Gwei
EJECTION_BALANCE: 16000000000
# 2**2 (= 4)
MIN_PER_EPOCH_CHURN_LIMIT: 4
# [customized] scale queue churn at much lower validator counts for testing
CHURN_LIMIT_QUOTIENT: 65536


# Fork choice
# ---------------------------------------------------------------
# 70%
PROPOSER_SCORE_BOOST: 70


# Deposit contract
# ---------------------------------------------------------------
# Ethereum Goerli testnet
DEPOSIT_CHAIN_ID: 1337 # Override for e2e tests
DEPOSIT_NETWORK_ID: 1337 # Override for e2e tests
# Configured on a per testnet basis
DEPOSIT_CONTRACT_ADDRESS: 0x1234567890123456789012345678901234567890


# Other e2e overrides
# ---------------------------------------------------------------
CONFIG_NAME: "end-to-end"
SLOTS_PER_EPOCH: 6
EPOCHS_PER_ETH1_VOTING_PERIOD: 2
MAX_SEED_LOOKAHEAD: 1
```

[e2e_config.zip](https://github.com/ConsenSys/web3signer/files/7927474/e2e_config.zip)

**Comment by usmansaleem** [comment]
@prestonvanloon thank you for reporting, I'll try to reproduce it locally and will report back today. We may need to update Teku libraries that do the network parsing.

**Comment by usmansaleem** [comment]
The error message is currently hidden in the stacktrace:
~~~
Detected unknown spec config entries: TERMINAL_TOTAL_DIFFICULTY,TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH,PROPOSER_SCORE_BOOST,TERMINAL_BLOCK_HASH,BELLATRIX_FORK_VERSION,BELLATRIX_FORK_EPOCH
~~~
Teku libraries web3signer currently using: `21.9.2`

Next step on my side is to test this file against latest Teku libraries. Also, try to print out detailed error message if possible.

**Comment by ajsutton** [comment]
Note that there was a recent change to rename the merge which affected these properties. Seems like web3signer is still expecting the pre-rename version.

**Comment by usmansaleem** [comment]
WIP https://github.com/ConsenSys/web3signer/pull/496

**Comment by prestonvanloon** [comment]
@usmansaleem any ETA on a resolution? 

We are blocked on adding web3signer support to our integration testing suite and that is blocking general release of web3signer support.

**Comment by usmansaleem** [comment]
@prestonvanloon within this week hopefully, I am resolving few ATs which are failing, also Teku will be releasing another version within this week, Web3Signer would be using those dependencies to parse network files.

**Comment by usmansaleem** [comment]
Fixed by #515.
